src/librawspeed/decoders/NefDecoder.cpp

```diff
@@ -76,7 +76,7 @@ RawImage NefDecoder::decodeRawInternal() {
   TiffEntry *offsets = raw->getEntry(STRIPOFFSETS);
   TiffEntry *counts = raw->getEntry(STRIPBYTECOUNTS);
 
-  if (data[0]->getEntry(MODEL)->getString() == "NIKON D100 ") { /**Sigh**/
+  if (data[0]->getEntry(MODEL)->getString() == "NIKON D100") { /**Sigh**/
```

Comment:
I guess the /**Sigh**/ was present because, originally, getString() did not trim, so the comparison had to me made against "NIKON D100 " (including the whitespace).

With the change in behaviour, the /**Sigh**/ can go away, too.

(Aside: is it safe to trim and then compare? The new code may change behaviour in case there ever is "NIKON D100 " and "NIKON D100", not that I would consider that likely)